test/e2e/flowaggregator_test.go

```diff
@@ -292,35 +315,17 @@ func checkPodAndNodeData(t *testing.T, record, srcPod, srcNode, dstPod, dstNode
 	}
 }
 
-func checkBandwidthFromRecord(t *testing.T, record, bandwidth string) {
-	// Split the record in lines to compute bandwidth
+func getTotalBytesFromRecord(t *testing.T, record string) float64 {
```

Comment:
same. The return type should be `uint64`

Reply:
Thanks, will address soon.